poncho/src/Makefile

```diff
@@ -0,0 +1,20 @@
+include ../../config.mk
+include ../../rules.mk
+
+PROGRAMS = poncho_package_analyze poncho_package_create poncho_package_run
+
+all:
+
+clean:
```

Comment:
The `clean` rule should remove derived programs in the build directory, not in the install directory.
This rule should be a no-op.